test/integration-tests/test-server/client.ts

```diff
@@ -0,0 +1,27 @@
+import { GraphQLClient } from 'graphql-request';
+import * as polarisPropertiesPath from './config/properties.json';
+
+export const url = `http://localhost:${polarisPropertiesPath.port}${
+    polarisPropertiesPath.endpoint
+}`;
+export async function graphqlRequest(data: string, headers: any, variables: any) {
```

Comment:
please use arrow functions here (all function in the file unless you have some special reason not to)